clean.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env sh
```

Comment:
Fine for now; but when we have CLI arguments overriding the config's action, we should remove this script.